### ISSUE TITLE
[epilepsy warning] block rendering rapidly appearing n disappearing when camera is moved on a 2 year old corrupted world [4.61 gigs]

### ISSUE THREAD

**Issue by NPC-Master3304** [jellysquid3/Sodium#1195]
i dont really know how 2 explain it, so heres a vid that sums it up:
https://youtu.be/8jN6CqPQ_Xc

but heres the interesting part, at 0, 0 its completely normal. 
https://youtu.be/rGKncdSmLjc

since this "only" happens on this world (as it doesn't happen 2 any other world I've used), I figured u would need the world to figure this out. it would be interesting to see what's going on here n y it only affects sodium like this. btw this started happening in 1.18. this hasn't happened in any other version where I've used sodium. I'm not really that surprised since 1.18 reworked chunks.
https://www.mediafire.com/file/olep201mvbtj869/corrupted.7z/file

to sum up this world, if i needed 2 test something, i would always go on this world (even if i needed a new world, i would just change the seed of that world, n then change it back afterwards). I've been doing it for the past 2 years. if i needed 2 go down to minecraft 1.8.9, i would. if i needed to go back in beta, then i would. i thought it would be unique to see a world thats been through so much corruption n other junk n see what it would end up looking like as it got worse n worse.

**Comment by Lolothepro** [comment]
Tried with Sodium alpha 6

**Comment by MeeniMc** [comment]
Can you repeat the bug with the latest release? This was an issue in dev builds that got fixed sometime in January.
